Gemfile

```diff
@@ -1,9 +1,9 @@
 source 'https://rubygems.org'
 
-ruby '2.3.3'
+ruby '2.4.0'
```

Comment:
Ruby 2.4.1 has been released. So please change 2.4.0 to  2.4.1.
https://www.ruby-lang.org/en/news/2017/03/22/ruby-2-4-1-released/

Also, please update `.ruby-version` file.
https://github.com/suzan2go/react-rails-redux-sample/blob/master/.ruby-version